.github/workflows/no-response.yml

```diff
@@ -0,0 +1,25 @@
+name: No Response
+
+on:
+  issue_comment:
+    types: [created]
+  schedule:
+    # Schedule at the end of every day
+    - cron: '0 0 * * *'
+
+jobs:
+  noResponse:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: lee-dohm/no-response@v0.5.0
+        with:
+          token: ${{ github.token }}
+          daysUntilClose: 7
+          responseRequiredLabel: waiting for response
+          responseRequiredColor: D28960
```

Comment:
Feels like we could just create the label on the UI, and then we don't need the hex code in the config here 😃

Reply:
The label is already created and you're right - we don't need it in the config